script/prep-travis-release.sh

```diff
@@ -92,7 +99,7 @@ BASENAME="rack"
 RACKBUILD="${BASENAME}${SUFFIX}"
 
 COMMIT=$(git rev-parse --verify HEAD)
-sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/util.go
+sed -i "s/var Commit =.*/var Commit = \"$COMMIT\"/" util/commit.go
```

Comment:
Sadly having kids and working in tech has meant that I've expanded more than I want to. Blame the :cookie: and :cake:.

I kid, I kid.

Why have one extra field to forget to update when we can update it in automation? This same replacement should happen in `script/build` as well and be idempotent (0.0.0-dev is what we're stuck on for the moment until we create a new git tag).